gradle.properties

```diff
@@ -4,7 +4,18 @@ org.gradle.caching=true
 org.gradle.priority=low
 
 # Gradle default is 256m which causes issues with our build - https://docs.gradle.org/current/userguide/build_environment.html#sec:configuring_jvm_memory
-org.gradle.jvmargs=-XX:MaxMetaspaceSize=512m
+# Also workaround https://github.com/diffplug/spotless/issues/834
```

Comment:
Somehow expected to have already added this by now. Noticed it since building on a new VM that I've only installed Java 17 on